app/components/customer-visible-icon/component.js

```diff
@@ -0,0 +1,7 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+  classNames: ["customer-visible-icon"],
+  attributeBindings: ["title"],
+  title: "This project's comments are visible to the customer"
+});
```

Comment:
This needs to be a glimmer component.

Reply:
Did you mean something like that?
`import Component from "@glimmer/component";

export default class CustomerVisibleIcon extends Component {
  classNames = ["customer-visible-icon"];
  attributeBindings = ["title"];
  title = "This project's comments are visible to the customer";
}`
That didn't render the component and crashed others.
Besides that, why does it need to be a glimmer component? I couldn't find another component using glimmer.

Reply:
Yeah, glimmer components don't render a DOM object. You need to create it in the template. See https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_outer-html for further infos

Reply:
Since those features can't be used yet (https://github.com/adfinis-sygroup/timed-frontend/pull/562#issuecomment-867644369) you can leave it as it is. However, please create an issue to fix it after the update is done.